doc/quick_start_guide/ffd/ffd.rst

```diff
@@ -22,14 +22,17 @@ Example designs
 ***************
 
 Demonstration
-===============================
+=============
 
-This is the far-field voice local command (FFD) example design with Sensory TrulyHandsfree™ (THF) speech recognition and local dictionary.
+This is the far-field voice local command (FFD) example design. Two examples are provided: both examples include speech recognition and a local dictionary. One example uses the Sensory TrulyHandsfree™ (THF) libraries, and the other one uses the Cyberon DSPotter™ libraries.
 
 When a wakeword phrase is detected followed by a command phrase, the application will output an audio response and a discrete message over |I2C| and UART.
 
+Sensory's THF software ships with an expiring development license. It will suspend recognition after 11.4 hours or 107 recognition events.
 
-Sensory's THF software ships with an expiring development license. It will suspend recognition after 11.4 hours or 107 recognition events; after which, a device reset is required to resume normal operation. To perform a reset, either power cycle the device or press the SW2 button.
+Cyberon's DSpotter™ software should run on a board with the correct OTP key programmed. If this is not the case, it will use an expiring development license. It will suspend recognition after 100 recognition events.
```

Comment:
Maybe say contact sales to discuss obtaining the OTP key?

Reply:
Can rephrase please - we should not mention OTP keys - The statement should be similar to the Sensory one - just state that the module ships with an evaluation licence that stops after 100 events.   

Production software runs on a special device - need to contact Cyberon or XMOS sale for information on that.

Reply:
@tomblackie-xmos, can you check if the sentences rephrased in https://github.com/xmos/sln_voice/pull/299/commits/545f89fbe559c7cc1fc0807e91d0d18aadcacbae are ok? I removed all the references to the OTP key.

Reply:
Added a comment on the commit.